lambdas/layer/.chalice/config.json.template.py

```diff
@@ -0,0 +1,12 @@
+from azul import config
+from azul.template import emit
+
+emit({
+    "version": "2.0",
+    "app_name": config.qualified_resource_name("azul_dependencies"),
```

Comment:
Doesn't this name the app `azul_azul_dependencies`?

Reply:
Something like that yes. It also doesn't matter since the app file is deleted, but I see value in changing it.